django/contrib/postgres/functions.py

```diff
@@ -1,4 +1,15 @@
 from django.db.models import DateTimeField, Func, UUIDField
+from django.db.models.fields import CharField
+
+
+class Jsonify(Func):
```

Comment:
Jsonify doesn't seem like the best name. If function is going to be in `contrib.postgres`, it should probably be documented and have its own tests.

Reply:
Does ``UpdateJSON`` make more sense ? I'll add the documentation and tests for the function appropriately.